build.rs

```diff
@@ -29,5 +30,69 @@ fn main() -> Result<(), Box<dyn Error>> {
     writeln!(f, "pub const USER_AGENT: &str = \"sentry-cli/{arch}\";")?;
     println!("cargo:rerun-if-changed=build.rs\n");
 
+    if platform == "darwin" {
+        println!("cargo:rerun-if-changed=native/swift/AssetCatalogParser");
+
+        env::set_current_dir("native/swift/AssetCatalogParser")
+            .expect("Failed to change to AssetCatalogParser directory");
```

Comment:
[change request]

I'd prefer we not change the working directory of the build script and rather use absolute paths, as I am not sure what side effects modifying the working directory could have